oozie-to-airflow/mappers/file_archive_mappers.py

```diff
@@ -50,22 +51,23 @@ def split_by_hash_sign(path: str) -> List[str]:
     return [path]
 
 
-# pylint: disable=too-few-public-methods
-class FileMixin:
-    """Mixin for file node"""
+class FileMapper:
```

Comment:
```suggestion
class FileHelper:
```
In the context of this project, the word "Mapper" refers to objects that convert Oozie's actions into Airflow tasks

Reply:
To me it's more general: it maps a part of an Oozie workflow to a part of an Airflow workflow. But I'm open to suggestions.
What do you think @potiuk ?

Reply:
Behaviour of your class is not compatible with your definition. This class converts the ooze node into Python dictionaries. It does not maps to Airflow objects.

Reply:
> to a part of an Airflow workflow
I didn't say "to Airflow objects"

Reply:
FileParser ? FileExtractor? FileReader ? FileConverter ?

Reply:
I don't like Helper . It says nothing

Reply:
I support every word that will not be the word "Mapper".  This is too confusing.

Reply:
Ok, I'll think of something. I agree mapper can be misleading as it's unlike the other mappers in the project. But similarly to Jarek, I don't like Helper.
Maybe FileExtractor? It extracts the path, after all.

Reply:
I like a FileExtractor.

Reply:
Names changed